nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestGenerateFlowFile.java

```diff
@@ -73,4 +78,30 @@ public void testDynamicPropertiesToAttributes() {
         generatedFlowFile.assertAttributeEquals("mime.type", "application/text");
     }
 
+    @Test
+    public void testExpressionLanguageSupport() {
+        TestRunner runner = TestRunners.newTestRunner(new GenerateFlowFile());
+        runner.setProperty(GenerateFlowFile.FILE_SIZE, "${random():mod(10):plus(1)}B");  // represents a range of 1-10 bytes, inclusive
```

Comment:
Instead of building the unit test against the complexities around the random function, it would be better to use a simple deterministic expression, avoiding the need for multiple runs.

Reply:
Modified to use `nextInt()` function. But still made 2 batch executions to confirm file size uniqueness (or not) based on `UNIQUE_FLOWFILES` setting. This is slightly out of scope, but the batch processing does not have any other unit test. 2 birds.